src/parseTools.mjs

```diff
@@ -44,7 +44,7 @@ export function processMacros(text, filename) {
 // Also handles #include x.js (similar to C #include <file>)
 export function preprocess(filename) {
   let text = read(filename);
-  if (EXPORT_ES6 && USE_ES6_IMPORT_META) {
+  if (EXPORT_ES6) {
     // `eval`, Terser and Closure don't support module syntax; to allow it,
```

Comment:
Potential follow up: I'm pretty sure closure supports module syntax now. Not sure about terser.

Reply:
I looked into this and will try to come up with followup.   For now is seems like the `eval` phase is the one were we still need this.